abb_irb6660_100_330_moveit_config/.setup_assistant

```diff
@@ -0,0 +1,11 @@
+moveit_setup_assistant_config:
```

Comment:
As moveit configs are usually application dependant they should probably not be part of this pull request.